en/getting_started/px4_basic_concepts.md

```diff
@@ -1,36 +1,81 @@
+
+## PX4 Autopilot {#autopilot}
+
+[PX4](http://px4.io/) is powerful open source autopilot *flight stack*. 
+
+Some of PX4's key features are:
+- Controls [many different vehicle frames/types](../airframes/airframe_reference.md), including: aircraft (multicopters, fixed wing aircraft and VTOLs), ground vehicles and underwater vehicles. 
+- Great choice of hardware for [vehicle controller](#vehicle_controller), sensors and other peripherals.
+- Flexible and powerful [flight modes](#flight_modes) and [safety features](#safety).
+
+
+## QGroundControl {#qgc}
+
+The Dronecode ground control station is called [QGroundControl](http://qgroundcontrol.com/).  You can use *QGroundControl* to load (flash) PX4 onto the [vehicle control hardware](flight_controller_selection.md), you can setup the vehicle, change different parameters, get real-time flight information and create and execute fully autonomous missions.
+
+*QGroundControl* runs on Windows, MacOS or Linux. Download and install it from [here](http://qgroundcontrol.com/downloads/). 
 
 ![QGC Main Screen](../../images/qgc_main_screen.jpg)
 
-## Heading and Directions
+## Vehicle Controller {#vehicle_controller}
 
-All the vehicles, boats and aircraft have a heading direction or an orientation based on their forward motion.
+PX4 was initially designed to run on Pixhawk-series controllers, but can now run on Linux computers and other hardware.
+You should select a board that suits the physical constraints of your vehicle, the activities you wish to perform, and of course cost.
 
-![Frame Heading](../../images/frame_heading.png)
+For more information see: [Flight Controller Selection](flight_controller_selection.md).
 
-It is important to know the vehicle heading direction in order to align the autopilot with the vehicle vector of movement. Despite it is not obvious, Multicopters have a heading despite they are symmetrical from all sides. Usually manufacturers use a colored props or colored arms to indicate the heading.
+## Sensors & Peripherals
 
-![Frame Heading TOP](../../images/frame_heading_top.png)
+PX4 uses sensors to determine vehicle state (needed for stabilization and to enable autonomous control). The system *minimally requires* a gyroscope, accelerometer, magnetometer (compass) and barometer. A GPS or other positioning system is needed to enable all automatic [modes](../getting_started/flight_modes.md#categories), and some assisted modes. Fixed wing and VTOL-vehicles should additionally include an airspeed sensor (very highly recommended).
 
-In our illustrations we will use red coloring for the front propellers of multicopter to show heading.
+For more information see: 
+* [Sensors](../getting_started/sensor_selection.md) 
+* [Peripherals](../peripherals/README.md)
 
-You can read in depth about heading in [Flight Controller Orientation](../config/flight_controller_orientation.md)
 
-## PX4 Connections
+## ESCs & Motors
```

Comment:
This is the section that kicked everything off.  Is this OK? Do we need to say anything more? Should we link to some quality ESC for purchase? Ditto for Motors? If so, which?

Reply:
I would not start to go into specific products, as the specific ESC or motor highly depends on the airframe size, total weight, LiPo cell numbers, desired flight duration, etc. It would quickly grow very large.

Reply:
Agreed. Thanks.